### PR TITLE
Adds a parsers section

### PR DESCRIPTION
**Pull request by FedericoPonzi**
Fixes issue #9.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,6 +71,10 @@ TLA+ is a formal specification and verification language to help engineers desig
 * [Practical TLA+](https://www.apress.com/gp/book/9781484238288)
 * [TLA+ in Practice and Theory](https://pron.github.io/posts/tlaplus_part1) (freely available)
 
+### Parsers
+* [tree-sitter-tlaplus](https://github.com/tlaplus-community/tree-sitter-tlaplus): A tree-sitter grammar for TLA⁺ and PlusCal
```
Comment (lemmy) [inline README.md:75]: SANY's and TLAPS' parser should probably be listed at the top.

Comment (FedericoPonzi) [reply README.md:78]: Sounds fair! I've just added them, not sure if maybe you have better links

**Review by lemmy — APPROVED** [review]

**Comment by lemmy** [comment]
Could have also been a subsection under `Tools`....

**Comment by FedericoPonzi** [comment]
Sorry, I meant to add it under tools. I'll quickly open a new pr

**Comment by lemmy** [comment]
Thanks!

**Pull request merged**